account_invoice_report_grouped_by_picking_optional_journal/models/account_move.py

```diff
@@ -0,0 +1,14 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit="account.move"
+
+    def lines_grouped_by_picking(self):
+        if not self.journal_id.show_group_by_picking_report:
```

Comment:
Add here singleton requirement, as original method does (https://github.com/OCA/account-invoice-reporting/blob/eeef78d3f5e279087485aa4b6db2a10d5b66eefe/account_invoice_report_grouped_by_picking/models/account_move.py#L61):
```suggestion
        self.ensure_one()
        if not self.journal_id.show_group_by_picking_report:
```